documentation/cdi/tutorial-cdi-instantiated-beans.asciidoc

```diff
@@ -1,36 +1,25 @@
 ---
-title: Components instantiated by the framework
+title: Using CDI Beans in Instantiated Components
 order: 2
 layout: page
 ---
 
-ifdef::env-github[:outfilesuffix: .asciidoc]
+= Using CDI Beans in Instantiated Components
 
-= Components instantiated by the framework
+When instantiated by Vaadin Flow, most components become managed beans. This means they are fully-fledged CDI contextual instances: all CDI features are usable when they are queried from the CDI `BeanManager` by the https://vaadin.com/directory/component/vaadin-cdi/[Vaadin CDI] add-on.
```

Comment:
```suggestion
When https://vaadin.com/directory/component/vaadin-cdi/[Vaadin CDI] is in use, most objects instantiated by framework become managed beans. Framework uses the CDI `BeanManager` to get references to beans. This means they are fully-fledged CDI contextual instances.
```

Trying to make this more compact and correct.

Reply:
Edited and committed your suggestion.

Reply:
Looks legit to me 👍